client/src/containers/RestaurantsPage/index.js

```diff
@@ -18,24 +19,26 @@ import getQueryParameters from '../../utils/getQueryParameters';
 import RenderView from './RenderView';
 import Filters from '../../components/Filters';
 
-function RestaurantsPage({ location: { search }, history }) {
+function RestaurantsPage({ location: { search }, history, locales }) {
   const start = parseInt(getQueryParameters(search, 'start'), 10) || 0;
   const orderby = getQueryParameters(search, 'orderby') || 'name';
   const range = 15;
 
   const setSearch = (where, nextStart) => {
     history.push({
-      search: `?category=${where.category}&district=${where.district}&start=${nextStart}`,
+      search: `?category=${where.category}&district=${where.district}&locale=${where.locale}&start=${nextStart}`,
     });
   };
 
   const getWhereParams = () => {
     const category = getQueryParameters(search, 'category') || 'all';
     const district = getQueryParameters(search, 'district') || '_all';
+    const locale = getQueryParameters(search, 'locale') || 'en';
```

Comment:
Do you have a way to access the default locale from the graphQL query?

Reply:
I don't think we can access the default locale from GraphQL

Reply:
WDYT @soupette?

Reply:
I don't know either to be honest.